pkg/network/kubevirt_ipam_controller_test.go

```diff
@@ -0,0 +1,45 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	osv1 "github.com/openshift/api/operator/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+)
+
+var _ = Describe("Testing kubevirt ipam controller", func() {
+	Context("Render KubevirtIpamController", func() {
+		conf := &cnao.NetworkAddonsConfigSpec{ImagePullPolicy: v1.PullAlways, Multus: &cnao.Multus{}, KubevirtIpamController: &cnao.KubevirtIpamController{}, PlacementConfiguration: &cnao.PlacementConfiguration{Workloads: &cnao.Placement{}}}
+		manifestDir := "../../data"
+		openshiftNetworkConf := &osv1.Network{}
+		clusterInfo := &ClusterInfo{SCCAvailable: false, OpenShift4: false}
```

Comment:
basically since all are false, those are the default values anyhow
so you can just create an empty struct
unless one of them is better to be explicit false, but this is not the case here

Reply:
DONE